spec/controllers/users/registrations_controller_spec.rb

```diff
@@ -1,22 +1,221 @@
-RSpec.describe Users::RegistrationsController do
+require 'rails_helper'
+
+RSpec.describe Users::RegistrationsController, type: :controller do
+  set_devise_mapping_to_user
```

Comment:
Because `RegistrationController` inherits from `Devise::RegistrationsController`, I've had to include some config for these specs [as per the Devise documentation](Devise::RegistrationsController).